src/main/java/com/google/api/codegen/util/ruby/RubyCommentReformatter.java

```diff
@@ -17,17 +17,44 @@
 import com.google.api.codegen.CommentPatterns;
 import com.google.api.codegen.util.CommentReformatter;
 import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
 import java.util.regex.Matcher;
 
 public class RubyCommentReformatter implements CommentReformatter {
   @Override
   public String reformat(String comment) {
```

Comment:
The list formatting logic depends on many variables being manipulated on the whole comment rather than just a line. The other transformations are discrete to a single line. The list formatting logic can't easily be separated into its own method.